wfl/Database/MongoDBManager.cs

```diff
@@ -63,9 +64,21 @@ private string GenerateCollectionName<T>()
         public async Task<T> GetEntityByIdAsync<T>(string id) where T: IEntity
         {
             IMongoCollection<T> collection = db.GetCollection<T>(GenerateCollectionName<T>());
-            IAsyncCursor<T> task = await collection.FindAsync(e => e.Id == id, null);
+            var filter = Builders<T>.Filter.Eq("Id", id);
```

Comment:
Using `nameof` will automatically rename the property if it ever changes in the future.